docs/user-guide/release-notes/2.12.x.rst

```diff
@@ -0,0 +1,11 @@
+=======================
```

Comment:
1 line needs to be added to the index file that indexes these release notes. Otherwise sphinx won't discover this file.